TL improvements (#353)
07th-mod/higurashi-console-arcs: Update/omot_015.txt

	ClearMessage();
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いつの頃から、こうなったのか忘れた。",
			NULL, "I forgot when I became like this.", Line_Normal);
	ClearMessage();

//rいや、思い出すのもわずらわしい。何がきっかけだったなんて思いつくだけでも十指に余るが、どこから落ち目を見出したのかなんて、考えたくもなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いや、思い出すのもわずらわしい。何がきっかけだったなんて思いつくだけでも十指に余るが、どこから落ち目を見出したのかなんて、考えたくもなかった。",

> 			NULL, "Or rather, I found it bothersome to remember. there were at least a dozen opportunities, but I did never want to think why I found myself corrupt.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Or rather, I found it bothersome to remember. There were at least a dozen potential triggers for it, but I never wanted to think about where my downfall began.", Line_Normal);
```

	ClearMessage();

//rそろいも揃って、今までろくなやつがいなかった。家族の面々の顔は思い出すだけでも憎たらしく、どうしているかも興味がない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そろいも揃って、今までろくなやつがいなかった。家族の面々の顔は思い出すだけでも憎たらしく、どうしているかも興味がない。",

> 			NULL, "Every and each guy around me is no good so far. I hated just remembering the faces of my family members, so I had no interest in how they have been.", Line_Normal);

Reviewer: ```suggestion
			NULL, "All these people around me, and there's not one good person among them. I hated just remembering the faces of my family members, so I had no interest in how they're doing.", Line_Normal);
```

	ClearMessage();

//r性別女は全て敵で、いつも食うか食われるか。寄ってくる男はつまらなさ全開で、横にはべらせてみても退屈以前に不愉快だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "性別女は全て敵で、いつも食うか食われるか。寄ってくる男はつまらなさ全開で、横にはべらせてみても退屈以前に不愉快だった。",

> 			NULL, "I have believed all women are enemies, and that it's always eat or be eaten. All the men who kissed up to me were totally boring, so it was rather annoying than boring to have a date with them.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I believed that all women are enemies, and that it's always eat or be eaten. All the men who kissed up to me were totally worthless, so dating them was more annoying than it was boring.", Line_Normal);
```

//rいや、思い出すのもわずらわしい。何がきっかけだったなんて思いつくだけでも十指に余るが、どこから落ち目を見出したのかなんて、考えたくもなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いや、思い出すのもわずらわしい。何がきっかけだったなんて思いつくだけでも十指に余るが、どこから落ち目を見出したのかなんて、考えたくもなかった。",
			NULL, "Or rather, I found it bothersome to remember. There were at least a dozen opportunities, but I did never want to think when I found myself corrupt.", Line_Normal);
	ClearMessage();

//rそろいも揃って、今までろくなやつがいなかった。家族の面々の顔は思い出すだけでも憎たらしく、どうしているかも興味がない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そろいも揃って、今までろくなやつがいなかった。家族の面々の顔は思い出すだけでも憎たらしく、どうしているかも興味がない。",

> 			NULL, "Every and each person around me is no good so far. I hated just remembering the faces of my family members, so I had no interest in how they have been.", Line_Normal);

Reviewer: ```suggestion
			NULL, "All these people around me, and there's not one good person among them. I hated just remembering the faces of my family members, so I had no interest in how they're doing.", Line_Normal);
```

//rいや、思い出すのもわずらわしい。何がきっかけだったなんて思いつくだけでも十指に余るが、どこから落ち目を見出したのかなんて、考えたくもなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いや、思い出すのもわずらわしい。何がきっかけだったなんて思いつくだけでも十指に余るが、どこから落ち目を見出したのかなんて、考えたくもなかった。",

> 			NULL, "Or rather, I found it bothersome to remember. There were at least a dozen opportunities, but I did never want to think about when I found myself corrupt.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Or rather, I found it bothersome to remember. There were at least a dozen potential triggers for it, but I never wanted to think about where my downfall began.", Line_Normal);
```

//r性別女は全て敵で、いつも食うか食われるか。寄ってくる男はつまらなさ全開で、横にはべらせてみても退屈以前に不愉快だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "性別女は全て敵で、いつも食うか食われるか。寄ってくる男はつまらなさ全開で、横にはべらせてみても退屈以前に不愉快だった。",

> 			NULL, "I have believed all women are enemies, and that it's always eat or be eaten. All the men who tried to seduce me were totally boring, so it was rather annoying than boring to have a date with them.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I believed that all women are enemies, and that it's always eat or be eaten. All the men who tried to seduce were totally worthless, so dating them was more annoying than it was boring.", Line_Normal);
```

//rちやほやと歯の浮くような言葉をぞろっと並べ立てても、そんなもの下心という、臭くて汚いどぶの中に放り投げているようなものだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ちやほやと歯の浮くような言葉をぞろっと並べ立てても、そんなもの下心という、臭くて汚いどぶの中に放り投げているようなものだ。",

> 			NULL, "Even if they churns out a lineup of affected language to flatter me, that's like throwing it inside the filthy, smelly gutter we call ulterior motive.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even if they churn out affectionate language to flatter me, that's like throwing it inside the filthy, smelly gutter of ulterior motives.", Line_Normal);
```

	ClearMessage();

//rだからいくらきらきらしていても、すぐに沈んで見えなくなってもう、うんざりだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だからいくらきらきらしていても、すぐに沈んで見えなくなってもう、うんざりだった。",

> 			NULL, "So no matter how brilliant that language is, it immediately sinks down and disappears. that made me feel absolutely disgusted.", Line_Normal);

Reviewer: ```suggestion
			NULL, "So no matter how vibrant their words are, they immediately sink down and disappear. That made me feel absolutely disgusted.", Line_Normal);
```

	ClearMessage();

	DrawScene("background/m_y4", 1000 );

//rぺっ、と唾を地面に吐き捨てる。さっきから口の中がきな臭くて、鬱陶しい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ぺっ、と唾を地面に吐き捨てる。さっきから口の中がきな臭くて、鬱陶しい。",

> 			NULL, "I spat down on the ground. The inside of my mouth had that burnt, annoying smell from earlier.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I spat down on the ground. The inside of my mouth had that burnt, irritating smell from earlier.", Line_Normal);
```

	ClearMessage();

//r水でもと辺りを見渡したが、こんな田舎道に自動販売機なんてあるわけがなく、それが余計にイラついて、もう何でもいいから怒鳴り散らしたい気分だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "水でもと辺りを見渡したが、こんな田舎道に自動販売機なんてあるわけがなく、それが余計にイラついて、もう何でもいいから怒鳴り散らしたい気分だった。",

> 			NULL, "I looked around for something to drink, but there were no vending machines on this backwater country road. That pissed me off even more, so I felt like I wanted to scream someone, no matter who are they.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I looked around for something to drink, but there were no vending machines on this backwater country road. That pissed me off even more, so I felt like I wanted to scream at someone, anyone.", Line_Normal);
```

//rだいたい今頃は、本当だったら穀倉の高級マンションに住処を移して、真新しいソファに腰掛けて酒でも飲んでいるはずだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だいたい今頃は、本当だったら穀倉の高級マンションに住処を移して、真新しいソファに腰掛けて酒でも飲んでいるはずだった。",

> 			NULL, "If things had played out properly, I'd be moving into a luxury apartment in Gogura right about now. I could've been lounging on a brand-new sofa, drinking alcohol in the first place.", Line_Normal);

Reviewer: ```suggestion
			NULL, "If things had played out properly, I'd be moving into a luxury apartment in Gogura right about now. I could've been lounging on a brand-new sofa, drinking alcohol.", Line_Normal);
```

			NULL, "\"Tch... What's this about family?!", Line_WaitForInput);
	ModPlayVoiceLS(4, 20, "ps3/s14/20/212100029", 256, TRUE);
	OutputLine(NULL, "　ナマ言ってんじゃねぇよドアホがぁっ！！」",
			NULL, " Don't say something so cocky, motherfucker!!\"", Line_Normal);
	ClearMessage();

//rあの竜宮とか言う冴えない中年の言葉を思い出して、辺り構わず怒鳴り散らす。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あの竜宮とか言う冴えない中年の言葉を思い出して、辺り構わず怒鳴り散らす。",

> 			NULL, "When I remembered what that middle-aged man called Ryuugu, I couldn't help but shout.", Line_Normal);

Reviewer: ```suggestion
			NULL, "When I remembered what that middle-aged man Ryuugu said, I couldn't help but shout.", Line_Normal);
```

	ClearMessage();

//rあれだけいい思いをさせてやったのに、突然やつとの連絡が一切取れなくなった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あれだけいい思いをさせてやったのに、突然やつとの連絡が一切取れなくなった。",

> 			NULL, "Even though I made him enjoy that much, he suddenly cut all contact with me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even after giving him such a good time, he suddenly cut all contact with me.", Line_Normal);
```

//rだいたい、テツなんて好きでもなんでもなかった。頭悪いし、ぎゃあぎゃあうるさいくせに気の小さいやつで、しかも働かずにゴロゴロしてるロクデナシで…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だいたい、テツなんて好きでもなんでもなかった。頭悪いし、ぎゃあぎゃあうるさいくせに気の小さいやつで、しかも働かずにゴロゴロしてるロクデナシで…。",
			NULL, "It's not like I even care about Tetsu anyway. He's a loud, touchy, and dimwitted guy who refuses to do any work...", Line_Normal);
	ClearMessage();

//r今回の取り分だって自分は大したことやってないくせに、山分けだとぬかしやがった。だから竜宮のオヤジから金をふんだくったら、やつとは手を切ろうと心に決めていたのだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "今回の取り分だって自分は大したことやってないくせに、山分けだとぬかしやがった。だから竜宮のオヤジから金をふんだくったら、やつとは手を切ろうと心に決めていたのだ。",

> 			NULL, "Even though he was hardly doing anything this time, he was so obsessed with taking his share as to claim to halve. So when I started bringing in money from that Ryuugu guy, I decided it was time to cut him off.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even though he was hardly doing anything this time, he was so obsessed with taking his share that he wanted half. So when I started bringing in money from that Ryuugu guy, I decided it was time to cut him off.", Line_Normal);
```

//rまさか。あれだけ入念に、見つかれば殺される危険を潜り抜けて手に入れた虎の子が、実は猫だったとでも言うのか？！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まさか。あれだけ入念に、見つかれば殺される危険を潜り抜けて手に入れた虎の子が、実は猫だったとでも言うのか？！",

> 			NULL, "Unbelievable. After we got over the crisis that we are slaughtered if they find us, we should've been able to obtain a tiger cub with so considerable care. But it was actually a normal cat, kidding me?!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Unbelievable. Even after such meticulous planning, knowing that we'd be slaughtered if they found us, we tiger cubs were no more than mere cats? Are you kidding me?!", Line_Normal);
```

> 			NULL, "Unbelievable. Since we got over the crisis that we are slaughtered if they find us after all that planning, we should've been able to obtain a tiger cub. But it was actually a normal cat, it's kidding me?!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Unbelievable. Even after such meticulous planning, knowing that we'd be slaughtered if they found us, we should've been able to obtain a nest egg! But it had already hatched? Are you kidding me?!", Line_Normal);
```

//r思い出すだけで腹が立つ。拳銃と葛西の眼光に何も言い返せず、恐怖に怯えてカッコ悪く逃げ出したことも今になっては、情けなくて悔しくて仕方がない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "思い出すだけで腹が立つ。拳銃と葛西の眼光に何も言い返せず、恐怖に怯えてカッコ悪く逃げ出したことも今になっては、情けなくて悔しくて仕方がない。",

> 			NULL, "Thinking back on it pissed me off. I couldn't say anything with Kasai wielding a gun in plain view. I was scared shitless and ran away. And it's nothing but shameful and mortifying now.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Thinking back on it pissed me off. I couldn't say anything with Kasai wielding a gun in plain view. I was scared shitless and ran away. And now, it's just shameful and mortifying.", Line_Normal);
```

> 			NULL, "Thinking back on it pissed me off. I couldn't say anything with Kasai wielding a gun in plain view. I was scared shitless and ran away. And it's now nothing but shameful and mortifying.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Thinking back on it pissed me off. I couldn't say anything with Kasai wielding a gun in plain view. I was scared shitless and ran away. And now, it's just shameful and mortifying.", Line_Normal);
```

> 			NULL, "Thinking back on it pissed me off. I couldn't say anything with Kasai wielding a gun in plain view. I was scared shitless and ran away ungainly. And now it's nothing but shameful and mortifying.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Thinking back on it pissed me off. I couldn't say anything with Kasai wielding a gun in plain view. I was scared shitless and ran away. And now, it's just shameful and mortifying.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "…なんてざまだ。ドブにまみれたような、爛れた生活から抜け出そうとした代償が、目の前のこれだ。身の程知らずが幸福を望んだ、神の天罰とでもどこぞの宗教家なら言うのだろうか。",

> 			NULL, "...What the hell is this? This filth in front of me is the price of trying to escape from this hideous life. Some religions would say that wanting to live above your position and find happiness is worthy of divine punishment?", Line_Normal);

Reviewer: ```suggestion
			NULL, "...What the hell is this? This filth in front of me is the price of trying to escape from this hideous life. What religion says that wanting to live above your position and find happiness is worthy of divine punishment?", Line_Normal);
```

//r何でもやってきた。女だてらにこのせち辛い世の中を生きていくために、身を切り売りするような思いを何度させられたことか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "何でもやってきた。女だてらにこのせち辛い世の中を生きていくために、身を切り売りするような思いを何度させられたことか。",

> 			NULL, "I made all my efforts I could. How many times have women thought about selling themselves to go on living in this harsh world?", Line_Normal);

Reviewer: ```suggestion
			NULL, "I did all I could. How many times have women thought about selling themselves to go on living in this harsh world?", Line_Normal);
```

//r診療所の警備なんて、たかが知れていた。たとえ失敗したところで、逮捕されて刑務所でおとなしくしていれば、数年で出所できる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "診療所の警備なんて、たかが知れていた。たとえ失敗したところで、逮捕されて刑務所でおとなしくしていれば、数年で出所できる。",
			NULL, "It was easy to imagine a clinic wouldn't have much security. And even if I'm arrested and sent to jail, I'll be out in just a few years as long as I keep quiet.", Line_Normal);
	ClearMessage();

//rその後のことは出てから考えるとしても、きっと興宮でもほとぼりは冷めている頃だろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その後のことは出てから考えるとしても、きっと興宮でもほとぼりは冷めている頃だろう。",

> 			NULL, "I won't consider what to do after being out until the release. Anyway, I'm sure they'll be cool down about the trouble in Okinomiya by then.", Line_Normal);

Reviewer: ```suggestion
			NULL, "No need to think about what to do after release until then. Anyway, I'm sure the trouble in Okinomiya will cool off by then.", Line_Normal);
```

//rその後のことは出てから考えるとしても、きっと興宮でもほとぼりは冷めている頃だろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その後のことは出てから考えるとしても、きっと興宮でもほとぼりは冷めている頃だろう。",

> 			NULL, "And even so I should consider what will happen after being out, I'm sure they'll be cool down about the trouble in Okinomiya.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And even thinking about what to do after I get out, I'm sure the trouble in Okinomiya will cool down.", Line_Normal);
```

//rリスク対効果を考えたら、公算が高いのがどっちかなんて明らかだ。どうしてあれだけの雁首そろえて、誰もそのことを言い出さなかったのか不思議でしょうがなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "リスク対効果を考えたら、公算が高いのがどっちかなんて明らかだ。どうしてあれだけの雁首そろえて、誰もそのことを言い出さなかったのか不思議でしょうがなかった。",

> 			NULL, "Considering the risk and rewards, it's clear what I should do. I wondered why none of us suggested it despite of numbers of crew.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Considering the risk and rewards, it's clear what I should do. I wondered why nobody ever suggested it, even though we had so many of us.", Line_Normal);
```

//r…あっさりと、ドアが開く。そしてその先には２階へと続く階段があり、ダンボールがその奥に山と詰まれてある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…あっさりと、ドアが開く。そしてその先には２階へと続く階段があり、ダンボールがその奥に山と詰まれてある。",

> 			NULL, "...The door surprisingly gently opened. And inside, I found a staircase leading to the second floor, with a mountain of cardboard boxes packed beside it.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...The door opened easily. And inside, I found a staircase leading to the second floor, with a mountain of cardboard boxes packed beside it.", Line_Normal);
```

			NULL, "\"The mountains brought forth a mouse... but it was actually a nasty rat.", Line_WaitForInput);
	ModPlayVoiceLS(4, 9, "ps3/s14/09/210900038", 256, TRUE);
	OutputLine(NULL, "面白そうだから泳がせてみたけど、どうやらただのコソ泥ってとこか。",

> 			NULL, " I let you alone because it looked fun, but you seems to be just a thief. ", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " I left you alone because it seemed amusing, but it looks like you're just a petty thief. ", Line_WaitForInput);
```

			NULL, "\"S-s-s-s-stop!", Line_WaitForInput);
	ModPlayVoiceLS(4, 20, "ps3/s14/20/212100053", 256, TRUE);
	OutputLine(NULL, "　そ、それ以上動くとっ！！」",

> 			NULL, " I-If you come any closer!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " D-don't come any closer!!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f6d9a8>鷹野</color>", NULL, "<color=#f6d9a8>Takano</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 9, "ps3/s14/09/210900058", 256, TRUE);
	OutputLine(NULL, "「既存の比較にもならない夢の最終兵器がね…。",

> 			NULL, "\"An dreamy ultimate weapon, beyond anything ever...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"An ultimate weapon, even beyond your wildest dreams...", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#3e5745>小此木</color>", NULL, "<color=#3e5745>Okonogi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 14, "ps3/s14/14/213400006", 256, TRUE);
	OutputLine(NULL, "「…大丈夫ですかね、こいつで？」",

> 			NULL, "\"...But is there no problem, with her?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...But isn't she going to be a problem?\"", Line_Normal);
```

			   NULL, "\"Nfu fu fu!", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps2/11/210700004", 540, TRUE);
		OutputLine(NULL, "　お変わりなくなんてひどいですなぁ！",

> 			   NULL, " You're so mean!", Line_WaitForInput);

Reviewer: ```suggestion
			   NULL, " What are you saying?", Line_WaitForInput);
```

	//r…休憩室で一服ついて戻る際、大石はふと顔を向けた一般入り口から、家族連れが入ってくるのに気づく。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "…休憩室で一服ついて戻る際、大石はふと顔を向けた一般入り口から、家族連れが入ってくるのに気づく。",

> 				NULL, "...On his way back after a short smoking in the break room, Ooishi suddenly noticed a family coming in through the public entrance.", Line_Normal);

Reviewer: ```suggestion
				NULL, "...On his way back after a short smoke in the break room, Ooishi suddenly noticed a family coming in through the public entrance.", Line_Normal);
```

	//r孫を持つってのは、こういうもんなんだろうなぁ。…そんなことを考えながら彼は、好色とは正反対のとろけそうな笑みでそれに応えた。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "孫を持つってのは、こういうもんなんだろうなぁ。…そんなことを考えながら彼は、好色とは正反対のとろけそうな笑みでそれに応えた。",

> 				NULL, "He started to wonder what it'd be like to have grandchildren. ...And as he did, he responded with a suitable smile of his own without any indecency.", Line_Normal);

Reviewer: ```suggestion
				NULL, "He started to wonder what it'd be like to have grandchildren. ...And as he did, he responded with a suitable smile of his own, without any indecency.", Line_Normal);
```

				NULL, "\"Na ha ha ha, don't mention it!", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1021", 256, TRUE);
		OutputLine(NULL, "　刑事の職務上、命を借りたことは忘れなくても、貸したことは水に流すのが礼儀ですからね～。",

> 				NULL, " All in the line of duty as a detective. Never forget that your life is borrowed, but it's common courtesy to forget you tried to give your life~.", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, " All in the line of duty as a detective. Never forget that your life is borrowed, but it's common courtesy to forgive and forget~.", Line_WaitForInput);
```

				NULL, "Akasaka found himself in a tight situation while wrestling the kidnapper, but he was able to escape with Ooishi's help.", Line_Normal);
		ClearMessage();

> 	//rそして、…はたと、気づく。あの時の奥さんのお腹の中には、この子がいたということに…。

Reviewer: ```suggestion
	//rそして、…はたと、気づく。kあの時の奥さんのお腹の中には、この子がいたということに…。
```

				NULL, "\"Thank you, ma'am!", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1028", 256, TRUE);
		OutputLine(NULL, "　というわけだから赤坂さん、せっかくのお心遣いです。",

> 				NULL, " Since she's so considerate, Akasaka-san.", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, " Since she's so considerate, Akasaka-san,", Line_WaitForInput);
```

		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1029", 256, TRUE);
		OutputLine(NULL, "今日はせいぜい、いろんな面子も入れて飲み明かしましょう！",

> 				NULL, " Then today, let's go all out and drink the night away, involving other men!", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, " for today, let's go all out and drink the night away with the guys!", Line_WaitForInput);
```

				NULL, "\"Thank you, ma'am!", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1028", 256, TRUE);
		OutputLine(NULL, "　というわけだから赤坂さん、せっかくのお心遣いです。",

> 				NULL, " Since she's this considerate, we should accept her kindness, Akasaka-san.", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, " Since she's so considerate, Akasaka-san,", Line_WaitForInput);
```

				NULL, " Since she's so considerate, Akasaka-san.", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1029", 256, TRUE);
		OutputLine(NULL, "今日はせいぜい、いろんな面子も入れて飲み明かしましょう！",

> 				NULL, " Then today, let's go all out and drink the night away with other guys!", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, " for today, let's go all out and drink the night away with the guys!", Line_WaitForInput);
```

			NULL, "\"During the years I've been waiting to retire, you've grown so much.", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s14/11/210700038", 256, TRUE);
	OutputLine(NULL, "いやいや、老兵は去るべきだと痛感します」",

> 			NULL, " It might be time for an old soldier simply to fade away, huh?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " It might be time for the younger generation to take over, huh?\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300032", 256, TRUE);
	OutputLine(NULL, "「……それで、実はこちらに来たら、古手梨花という少女に会いたいと思っていたんです」",

> 			NULL, "\"Anyway, the truth is that I want to meet with Rika Furude if I have an opportunity to visit here.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Anyway, I want to meet with Rika Furude.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "古手梨花はかつて赤坂に、雛見沢で５年連続して起こるという奇怪な事件を予言している。その予言の中の１つ、確か３年目の予言に、彼女の両親が殺されるというものが含まれていたはずだ。",

> 			NULL, "Rika Furude had predicted that there would be an mysterious incident in Hinamizawa every year for the following five years. Her parents, according to her, were supposed to be killed on the third year running.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Rika Furude had predicted that there would be an incident in Hinamizawa every year for the following five years. Her parents, according to her, were supposed to be killed on the third year running.", Line_Normal);
```

//r…そうだ、私はこれを確かめるために来たんじゃないか…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…そうだ、私はこれを確かめるために来たんじゃないか…。",

> 			NULL, "...of course. I had, after all, come here to confirm it.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...He had, after all, come here to confirm it.", Line_Normal);
```

> 			NULL, "...Of course. I had, after all, come here to confirm it.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...He had, after all, come here to confirm it.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300042", 256, TRUE);
	OutputLine(NULL, "「つ、続けて確認します。",

> 			NULL, "\"I'll con-continue.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I'll continue.", Line_WaitForInput);
```

> 			NULL, "\"I-I'll continue.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I'll continue.", Line_WaitForInput);
```

			NULL, "\"I think... let's see... I know.", Line_WaitForInput);
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300045", 256, TRUE);
	OutputLine(NULL, "多分、サトコさんという、女性の方だと思います。",

> 			NULL, " Satoko-san, it was. I think it's a name of a lady.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Satoko-san, it was. I think she's a little girl.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300046", 256, TRUE);
	OutputLine(NULL, "その方のご両親が転落でお亡くなりになるということは？」",

> 			NULL, " Did her parents die in a fall accident?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Did her parents die in an accident?\"", Line_Normal);
```

			NULL, "\"I think... let's see... I know.", Line_WaitForInput);
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300045", 256, TRUE);
	OutputLine(NULL, "多分、サトコさんという、女性の方だと思います。",

> 			NULL, " Satoko-san, it was. I think it's name of a lady.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Satoko-san, it was. I think she's a little girl.", Line_WaitForInput);
```

			NULL, " None of those incidents were made public because of pressure from the locals,", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s14/00/213000013", 256, TRUE);
	OutputLine(NULL, "東京にいながら、よくそれだけ知れたもんだわい」",

> 			NULL, " so I'm surprised at how much you know about them in Tokyo.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " so I'm surprised at how much you know about them.\"", Line_Normal);
```

> 			NULL, " so I'm surprised at how much you know about them although you live in Tokyo.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " so I'm surprised at how much you know about them.\"", Line_Normal);
```

> 			NULL, " so I'm surprised at how much you know about them even though you live in Tokyo.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " so I'm surprised at how much you know about them.\"", Line_Normal);
```

			NULL, "\"Sh-Shit, you're right.", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s14/11/210700062", 256, TRUE);
	OutputLine(NULL, "赤坂さん、どうしてそんなことまでご存じなんですか」",

> 			NULL, " Akasaka-san, why do you know so much?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Akasaka-san, how do you know so much?\"", Line_Normal);
```

//rだが、自分で口にしている内容が当たっていて呆然とした表情を見せる赤坂の妙な様子と、わかるはずもない今年の祟りの犠牲者の名前を口にするのを見るに至り、彼らの酔いは完全に吹き飛ぶのだった…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だが、自分で口にしている内容が当たっていて呆然とした表情を見せる赤坂の妙な様子と、わかるはずもない今年の祟りの犠牲者の名前を口にするのを見るに至り、彼らの酔いは完全に吹き飛ぶのだった…。",

> 			NULL, "However, not only was he dumbfounded to know what he spoke about is a fact, but he also named the next victim they never knew. Ooishi and his coworkers had sobered up entirely.", Line_Normal);

Reviewer: ```suggestion
			NULL, "However, not only did he know about them, but he also named the next victim. Ooishi and his coworkers had sobered up entirely.", Line_Normal);
```

> 			NULL, "However, not only was he dumbfounded to know what he said is a fact, but he also named the next victim none of them knew. Seeing that strange scene, Ooishi and his coworkers had sobered up entirely.", Line_Normal);

Reviewer: ```suggestion
			NULL, "However, not only did he know about them, but he also named the next victim. Ooishi and his coworkers had sobered up entirely.", Line_Normal);
```

> 			NULL, "However, not only was he dumbfounded to know what he told is a fact, but he also named the next victim none of them knew. Seeing that strange scene, Ooishi and his coworkers had sobered up entirely.", Line_Normal);

Reviewer: ```suggestion
			NULL, "However, not only did he know about them, but he also named the next victim. Ooishi and his coworkers had sobered up entirely.", Line_Normal);
```

//r大石はその後に、そんな馬鹿な、と言葉を続けるつもりだったが、…あまりのことに、あんぐりと開けた口からそれ以上の言葉が出ることはなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大石はその後に、そんな馬鹿な、と言葉を続けるつもりだったが、…あまりのことに、あんぐりと開けた口からそれ以上の言葉が出ることはなかった。",

> 			NULL, "Ooishi actually tried to say, \"What nonsense!\" But it was so surprising that he was left speechless.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Ooishi couldn't believe what he was hearing. He was left speechless.", Line_Normal);
```

> 			NULL, "Ooishi actually tried to say, \"What nonsense!\" But he was so surprised that he was left speechless.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Ooishi couldn't believe what he was hearing. He was left speechless.", Line_Normal);
```

			NULL, " Rika Furude told me this in June 1978.", Line_WaitForInput);
	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300065", 256, TRUE);
	OutputLine(NULL, "…ただ、当時の私はそれを重要な話とは思わず、それを大石さんに伝えませんでした。",

> 			NULL, " But at the time, I didn't think it was important information, so I didn't tell you about it.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " But at the time, I didn't think anything of it, so I didn't tell you about it.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>鑑識</color>", NULL, "<color=#f5e6d3>Forensic Investigator</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/213000018", 256, TRUE);
	OutputLine(NULL, "「い…いや、それは無理もないぞい。",

> 			NULL, "\"N-No, that's natural.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"N-No, you didn't know.", Line_WaitForInput);
```

> 			NULL, "\"N-No, That's natural.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"N-No, you didn't know.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s14/11/210700078", 256, TRUE);
	OutputLine(NULL, "それで３年目の祟りに遭ったというのが定説です。",

> 			NULL, " That must be the reason why the curse fell upon them in the third year.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " That might be the reason why the curse fell upon them in the third year.", Line_WaitForInput);
```

//r大石は無言で、ゆっくりと頷いた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大石は無言で、ゆっくりと頷いた。",

> 			NULL, "Ooishi nodded slowly without saying anything.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Ooishi nodded slowly.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s14/11/210700081", 256, TRUE);
	OutputLine(NULL, "「…私が一緒じゃまずかろうと思いますので今はお任せしますが、古手梨花の話は、どうやら私にとっても重要なものになりそうです」",

> 			NULL, "\"...I don't want to bother you, so I'm leave it to you now. But what Rika Furude said is important to me, too.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...It's not that I don't want your help, but what Rika Furude said is important to me.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s14/11/210700087", 256, TRUE);
	OutputLine(NULL, "「悔しいですが、そういうことになるでしょうな」",

> 			NULL, "\"Yeah, unfortunately I guess.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Yeah, I guess.\"", Line_Normal);
```
